apps/estuary/test/unit/test_helper.exs

```diff
@@ -1 +1,4 @@
-ExUnit.start()
+Application.ensure_all_started(:mox)
+Application.ensure_all_started(:dead_letter)
```

Comment:
are these necessary?

Reply:
Yes, they are necessary, since we are doing the unit test with no start but we still require mox and dead_letter